ssl/tls13_enc.c

```diff
@@ -107,6 +107,7 @@ int tls13_hkdf_expand_ex(OSSL_LIB_CTX *libctx, const char *propq,
                          const unsigned char *data, size_t datalen,
                          unsigned char *out, size_t outlen, int raise_error)
 {
+    // TODO: Should we support DTLS labels from here? This interface is only used by QUIC.
```

Comment:
I'd say there is no need at this point. Maybe just write a comment saying we only support TLSv1.3 and not DTLSv1.3 in this function (nit: use `/* ... */` not `//` style comments.)